build.gradle

```diff
@@ -0,0 +1,42 @@
+buildscript {
+  ext {
+    dep = [
+            assertj               : "org.assertj:assertj-core:1.7.0",
```

Comment:
1) Need to update the versions.
2) Would be great to move versions into a separate map, similar to [Okio's build.gradle](https://github.com/square/okio/blob/master/build.gradle)
3) Aligning by `:` symbols is undesirable since it may cause all lines to be reformatted if an entry with a longer key is added, leading to unnecessary diffs. Stick to the following format: `key : "value"`.